ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/management-tech-support.j2

```diff
@@ -0,0 +1,41 @@
+{# Management Tech-Support #}
+{% if management_tech_support is arista.avd.defined %}
+
+## Management Tech-Support
+{%     if management_tech_support.policy_show_tech_support is arista.avd.defined %}
+
+### Policy
+{%         if management_tech_support.policy_show_tech_support.exclude_commands is arista.avd.defined %}
+
+#### Exclude Commands
+
+| Command | Type |
+| ------- | ---- |
+{%             if management_tech_support.policy_show_tech_support.exclude_commands is arista.avd.defined %}
+{%                 for exclude_command in management_tech_support.policy_show_tech_support.exclude_commands %}
+| {{ exclude_command.command }} | {{ exclude_command.type | arista.avd.default('text') }} |
+{%                 endfor %}
+{%             endif %}
+{%         endif %}
+
+{%         if management_tech_support.policy_show_tech_support.include_commands is arista.avd.defined %}
+
+#### Include Commands
+
+| Command |
+| ------- |
+{%             if management_tech_support.policy_show_tech_support.include_commands is arista.avd.defined %}
+{%                 for include_command in management_tech_support.policy_show_tech_support.include_commands %}
+| {{ include_command.command }} |
```

Comment:
Please but both the include and exclude commands in backtics like `my included command`. This is to avoid a situation like the markdown we have in the molecule:
![image](https://user-images.githubusercontent.com/48674677/160155478-8ce2f7eb-b982-40a5-abf1-a1328d6c3029.png)

Reply:
the fix didn't work. It still breaks the line into two "cells" in the table.
We had the same problem in `as-path.j2` where we ended up adding ` | replace('|', '\|')` to the printout. You can remove the backticks again if you like.

Reply:
done